src/components/Footer.jsx

```diff
@@ -11,23 +11,20 @@ function Footer() {
             <img
               src="https://freepikpsd.com/wp-content/uploads/2019/10/adresse-icone-png-6-Transparent-Images.png"
               alt="adresse"
-              width="25px"
```

Comment:
I'm ashamed to see I let it pass the first time >_<
But thanks for fixing this !